src/v/cloud_topics/level_zero/pipeline/base_pipeline.h

```diff
@@ -165,6 +165,7 @@ class base_pipeline {
         for (auto& f : _filters) {
             f.cancel();
         }
+        remove_requests_for_shutdown();
```

Comment:
why is this not called in the shutdown method :)

Reply:
oh, yeh i meant to rename it to `for_stop`